tests/zenko_tests/e2e_config/accounts.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import logging
+
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+_log = logging.getLogger("end2end configuration")
+
+
+def create_account_secret(name, accessKey, secretKey, namespace="default"):
+    """
+    """
+    _log.info("creating account secret")
+    core = client.CoreV1Api()
+    secret = client.V1Secret(
+        api_version="v1",
+        metadata=client.V1ObjectMeta(
+            name=name,
+            labels={
+                "type": "end2end",
+            },
+        ),
+        string_data={
+            "accessKey": accessKey,
+            "secretKey": secretKey,
+        },
+    )
+
+    res = core.create_namespaced_secret(namespace, body=secret)
+
+    _log.info("created account secret")
+
+
+def create_account(client, uuid, account_name, namespace="default"):
+    """
+    Creates a user account and save its accessKey and secretKey as k8s secret
+    """
+    try:
+        User_V1 = client.get_model('user-v1')
+        u = User_V1(userName=account_name)
+
+        res = client.ui_facing\
+            .createConfigurationOverlayUser(user=u, uuid=uuid)\
+            .response()\
+            .result
```

Comment:
Instead of using a backslash, you can wrap in parenthesis.

```suggestion
        res = (
            client.ui_facing
            .createConfigurationOverlayUser(user=u, uuid=uuid)
            .response()
            .result
        )
```

Reply:
wasn't sure about doing that, I had thought that make it a tuple, probably changed in python3

Reply:
tuples have to have commas, so `(0,)` is a tuple but `(0)` is just 0. Even without parenthesis, it can be a tuple so `return 0,` returns a single item tuple. Funny errors from that one...